docs/virtual-hosts/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Virtual Hosts
+intro: Learn how to configure virtual hosts to register domains and assign IPs for application access
```

Comment:
Nope. At the momento Devopness does not `register domains`, so this text here is misleading to the users.

```suggestion
intro: Learn how to configure virtual hosts to have one or more applications publicly accessible on the internet by IP address or domain name
```